pdns/rfc2136handler.cc

```diff
@@ -204,10 +206,10 @@ uint PacketHandler::performUpdate(const string &msgPrefix, const DNSRecord *rr,
         }
         if (updateTTL > 0) {
           di->backend->replaceRRSet(di->id, rr->d_name, rrType, rrset);
-          g_log<<Logger::Notice<<msgPrefix<<"Replacing record "<<rr->d_name<<"|"<<rrType.toString()<<endl;
+          g_log<<Logger::Notice<<msgPrefix<<"Updating TTLs for "<<rr->d_name<<"|"<<rrType.toString()<<endl;
```

Comment:
I'm a bit confused by that change, since it looks like we might be updating more than the TTLs in that case, or am I missing something?

Reply:
It is, painfully, a full DELETE+INSERT, but the net change really should be just the TTLs. If that's not true, I missed something.

Reply:
You are not, I was! It took me several tries to understand exactly what `rr` and `rrset` are at this point, sorry.